examples/sparseserver-ui/README.md

```diff
@@ -86,3 +86,19 @@ Visit `http://localhost:8501` in your browser to view the demo.
 - Ubuntu 20.04.4 LTS
 - Python 3.8.10
 </samp>
```

Comment:
would add </samp> at the end of the file to maintain style

Reply:
Will do, cheers